openssl/src/conf.rs

```diff
@@ -6,17 +7,25 @@ use error::ErrorStack;
 pub struct ConfMethod(*mut ffi::CONF_METHOD);
 
 impl ConfMethod {
+    /// Retrieve handle to the default OpenSSL configuration file processing function.
+    ///
+    /// # OpenSSL Implementation Details
+    ///
+    /// `NCONF` stands for "New Conf", as described in crypto/conf/conf_lib.c. This is
```

Comment:
I don't think we expose the name `NCONF` anywhere, right?

Reply:
Correct, this is an artifact of someone coming in completely unfamiliar with the OpenSSL conf module. My first question when looking at the `ffi` calls was "What does `NCONF` even stand for?" There is no documentation for the module, so it took me a while of digging in the source code to even figure that out.

I had to actually read part of a Google Book, to figure out what OpenSSL was actually exposing here.

This could be changed to a non-doc comment as it's more useful for those actually looking at the implementation and fixing/extending it.

Thoughts?

Reply:
Ah gotcha, that makes sense! I'd change this to a normal comment, then, since it's something that's relevant to someone reading the source but not the API docs.

Reply:
Done.